scripts/utils/variables.py

```diff
@@ -31,7 +31,8 @@
 KAFKA_TOPIC = os.getenv('KAFKA_TOPIC')
 KAFKA_GROUP_PREFIX = os.getenv('KAFKA_GROUP_PREFIX', '')
 
-WEB3_CHAIN_ID = web3.eth.chain_id
+# Should be reinitialized after brownie pre-script
+WEB3_CHAIN_ID = 1
```

Comment:
maybe we should pass it through env variables and then check that each provider in the MultiHTTPProvider matches this chain_id. @infloop did so in the js provider